lib/pub_sub/publisher.rb

```diff
@@ -1,28 +1,28 @@
 module PubSub
   class Publisher
     class << self
-      def publish(message, async: false)
+      def publish(message, async: false, custom_topic: PubSub.service_identifier)
```

Comment:
I see that it starts with a default. here :point_up_2: . Do we want to call this `custom_topic` seems like topic should suffice.